extensions-core/s3-extensions/src/main/java/org/apache/druid/storage/s3/S3DataSegmentKiller.java

```diff
@@ -150,7 +151,19 @@ private boolean deleteKeysForBucket(
             s3Bucket,
             keysToDeleteStrings
         );
-        s3Client.deleteObjects(deleteObjectsRequest);
+        RetryUtils.retry(
+            () -> {
+              try {
+                s3Client.deleteObjects(deleteObjectsRequest);
+                return null;
+              }
+              catch (Exception e) {
```

Comment:
I don't understand the point of try/catch around it, it doesn't seem to do anything?

Reply:
Thanks, fixed